DESCRIPTION

```diff
@@ -2,7 +2,7 @@ Package: annoFuse
 Type: Package
 Title: Annotation And Filtering Analysis For Fusion Calls
 Version: 0.2
-Authors@R: c(person("Krutika", "Gaonkark", role = c("aut", "cre"), 
+Authors@R: c(person("Krutika", "Gaonkar", role = c("aut", "cre"), 
     email = "gaonkark@chop.email.edu"),
```

Comment:
```suggestion
    email = "gaonkark@chop.email.edu"),
    comment = c(ORCID = "0000-0003-0838-2405")))
```
can you add other things, such as github handles? @federicomarini ?

Reply:
@kgaonkar6 I think we should add @komalsrathi as an author, as we started this with some of her initial code?

Reply:
AFAIK only ORCID can be added as comment, as for now

Reply:
As for @komalsrathi: sure, no problem!